test/static/js/odin_server.js

```diff
@@ -1,50 +1,143 @@
 api_version = '0.1';
 
 $( document ).ready(function() {
-
-    update_api_version();
-    update_api_adapters();
-    poll_update()
+    show_code_list();
+    clear_list();
+    clear_email();
 });
 
-function poll_update() {
-    update_background_task();
-    setTimeout(poll_update, 500);   
+function handle_form_test(){
+
+    var task_access_code = $('#input_test').val();
+    $('#displayer').html(task_access_code);
+    $('#displayer').css('color', 'green');
+    $('#validity').html("your code is valid");
+
 }
 
-function update_api_version() {
+function bad_code_response(err_msg){
+    var input_code = $('#input_test').val();
+    $('#displayer').html(input_code).val();
+    $('#displayer').css('color', 'red');
+    $('#validity').html(err_msg);
+}
 
-    $.getJSON('/api', function(response) {
-        $('#api-version').html(response.api);
-        api_version = response.api;
+function check_code_validation() {
+    var input_code = $('#input_test').val();
+    if (/^([A-N)]|[P-Z]|[1-9]){7}$/.test(input_code))
+    {
+        $.ajax({
+            type: "PUT",
+            url: '/api/' + api_version + '/timeslice',
+            contentType: "application/json",
+            data: JSON.stringify({'add_access_code': input_code})
+        })
+        .done(function(json){
+            console.log("calling handle_form_test")
```

Comment:
There a quite a few `console.log()` calls through the JS which should probably be removed when you're happy with the code